### DISCUSSION TITLE
Beatmap doesn't get separated regardless of their defined artist and title in the difficulty's .osu file if they are in the same set

### DISCUSSION THREAD

**Discussion by timelessnesses** [ppy/osu#30020] [Q&A]
![image](https://github.com/user-attachments/assets/7e0ca84a-3203-41e6-ade5-69d075a3cf8d)  
![image](https://github.com/user-attachments/assets/91ddc4ea-81a1-45f9-8e2c-bb268427b6d9)  
The entire beatmap set's artist and title name will be changed if one of the difficulty in the set provided  different artist/title name instead of separate the difficulty that has different artist/title to its own set

**Comment by timelessnesses** [comment]
the `shitting` difficulty is imported from osu! client which the `shitting` difficulty is placed in the same folder as the FANCY - What is Love set and `shitting` difficulty also points to same audio file as the rest of the set too

**Comment by bdach** [comment]
How have you even entered this state? This smells of manual file editing on stable and then importing said manually broken set into lazer.

I don't think we should support this, this sort of metadata change thing is unrankable? If you wanted to do that the set should be named appropriately and the artist should be "Various Artists" or whatever.

**Reply by timelessnesses** [reply]
So I placed the difficulty file from other song and I put it in the new set, then I imported it through osu!lazer. And yes this kind of stuff is unrankable but some beatmaps does this
